src/host/VtIo.cpp

```diff
@@ -262,10 +195,10 @@ bool VtIo::IsUsingVt() const
     // We need both handles for this initialization to work. If we don't have
     //      both, we'll skip it. They either aren't going to be reading output
     //      (so they can't get the DSR) or they can't write the response to us.
-    if (_lookingForCursorPosition && _pVtRenderEngine && _pVtInputThread)
+    if (_pVtInputThread && _pVtInputThread->IsLookingForDSR())
     {
-        LOG_IF_FAILED(_pVtRenderEngine->RequestCursor());
-        while (_lookingForCursorPosition && _pVtInputThread->DoReadInput())
+        WriteUTF8("\x1b[6n"); // Cursor Position Report (DSR CPR)

```

Comment:
Okay, I have a dumb idea. Should (or can?) we have a macro that _literally compiles down to `WriteUTF8` or whatever_ called `WriteActualVTOrSomething` so we can **find every place we write direct hardcoded VT**?

Reply:
We could but how would you ensure that it's only used for VT?
FWIW the number of places that will call these `Write*()` functions will remain fixed in the future, because we don't plan on adding more console API functions.

Reply:
(Resolving, because all the sequences are now within VtIo.)